new-site/src/docs/elements/components/card/code.mdx

```diff
@@ -26,29 +23,39 @@ import Playground from '../../../../components/Playground';
 ```
 
 ```html
-<div>
-  <div class="hds-card"></div>
-  <div class="hds-card hds-card--border"></div>
-</div>
+  <div class="hds-card">
```

Comment:
There's some problems with rendering of these cards:

<img width="1006" alt="image" src="https://user-images.githubusercontent.com/2777633/155750531-2b80e30d-0ad7-4303-b7c0-66de7a1ffa48.png">

Reply:
- [ ] add wrapper around elements